src/client/cypress/integration/Demo/trade.js

```diff
@@ -0,0 +1,32 @@
+import { Given, When, Then, And} from 'cypress-cucumber-preprocessor/steps'
```

Comment:
Please use `.ts`

Reply:
as discussed, continuing with .js only for the time being